factcast-snapshotcache-redisson/src/test/java/org/factcast/core/snap/redisson/RedissonSnapshotCacheTest.java

```diff
@@ -166,4 +172,19 @@ private void sleep(long millis) {
       Thread.sleep(millis);
     }
   }
+
```

Comment:
There are tests missing that validate the use of the codec when reading & writing

Reply:
I added one test that checks both. Not sure if it is sufficient though.